pxr/usd/sdf/textParserContext.cpp

```diff
@@ -25,20 +25,16 @@
 #include "pxr/pxr.h"
 #include "pxr/usd/sdf/textParserContext.h"
 
-PXR_NAMESPACE_OPEN_SCOPE
```

Comment:
There was some refactoring done to make this object useable from both lexers.  If I remember correctly, removing the namespace here made the resulting `.lex` files much simpler (because they don't live in the namespace) but I'd have to back and verify.